tests/libvmi/storage.go

```diff
@@ -96,3 +116,122 @@ func newContainerVolume(name, image string) v1.Volume {
 		},
 	}
 }
+
+func VerifyVolumeAndDiskVMAdded(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine, volumeNames ...string) {
+	nameMap := make(map[string]bool)
+	for _, volumeName := range volumeNames {
+		nameMap[volumeName] = true
+	}
+	log.Log.Infof("Checking %d volumes", len(volumeNames))
+	Eventually(func() error {
+		updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Get(vm.Name, &metav1.GetOptions{})
```

Comment:
Actions like this should not occur in the package, it takes it to a different area.
We once considered adding operations, but we have not done that yet or planned how it is best to add it here.